compiler/rustc_privacy/src/lib.rs

```diff
@@ -1407,8 +1401,12 @@ impl DefIdVisitor<'tcx> for TypePrivacyVisitor<'tcx> {
     fn tcx(&self) -> TyCtxt<'tcx> {
         self.tcx
     }
-    fn visit_def_id(&mut self, def_id: DefId, kind: &str, descr: &dyn fmt::Display) -> bool {
-        self.check_def_id(def_id, kind, descr)
+    fn visit_def_id(&mut self, def_id: DefId, kind: &str, descr: &dyn fmt::Display) -> ControlFlow<(), ()> {
+        if self.check_def_id(def_id, kind, descr) {
```

Comment:
these are very common. Maybe we should have a `ControlFlow::break_if(some_bool)` and `ControlFlow::continue_if(some_bool)` for convenience? Or will that become infeasible with types other than `()`?

Reply:
yeah, I think adding these methods make sense. I am not yet sure how to be generalize them to deal with types other than `ControlFlow<(), ()>`

Reply:
There has been [almost no thought](https://github.com/rust-lang/rust/issues/75744#issuecomment-679250842) put into what methods the `ControlFlow` type should have, so if you come up with reasonable ones definitely add them.  It used to be an internal implementation detail, so doesn't even have the usual combinators.

Reply:
@scottmcm I'd say definitely `is_break` and `is_continue` (name to be bikeshed of course). I'll file a PR